platform/android/os_android.h

```diff
@@ -154,6 +155,8 @@ class OS_Android : public OS_Unix {
 	VideoPauseFunc video_pause_func;
 	VideoStopFunc video_stop_func;
 	SetKeepScreenOnFunc set_keep_screen_on_func;
+	
+	power_android power_manager;
```

Comment:
That's corrected. I just pushed the fix, but I can't affirm android will build, I'm almost certain there are other problems, especially in ``platform/android/power_android.cpp``'s method ``int Android_JNI_GetPowerInfo(int* plugged, int* charged, int* battery, int* seconds, int* percent)``